site/_posts/2017-08-29-where-have-all-the-cow-wranglers-gone.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Last Week in ManageIQ: Where Have All The Cow Wranglers Gone"
+author: allenbw
+date: 2017-08-29
+comments: true
+published: true
+tags: LWIMIQ
+---
+Hi Friends!  [Allen](https://github.com/allenbw) here, with news to share!
+Since [we last talked](http://manageiq.org/blog/2017/08/back-to-basics/) much has happened, two weeks of on goings are due for discussion.
+A few quick stats before we dig in:
+* No new repositions this time around!
+* ~ 447 PRs closed across 133 ManageIQ Repos (between 08/07 to 08/21)
+* In descending order, the top four repositories by pull request activity:
+    * [UI Classic](https://github.com/ManageIQ/manageiq-ui-classic)
+    * [ManageIQ](https://github.com/ManageIQ/manageiq)
+    * [Integration Tests](https://github.com/ManageIQ/integration_tests)
+    * [ManageIQ Docs](https://github.com/ManageIQ/manageiq_docs)
+* Downstream version updated from 4.5 to 4.6 [here](https://github.com/ManageIQ/manageiq_docs/pull/491)
+Okay okay so let's get into it.
+## Featured
+Oh my! Without a repo fork to feature, lets turn our attention to a hotly discussed and highly welcomed update to the `Container Image` list view.
+Now it includes a [`Number of Containers` column](https://github.com/ManageIQ/manageiq/pull/15741). Those little usability features just make all the difference.
+## Improved
+### Additional Class Methods
+It was requested [in ManageIQ/manageiq-api#3](ManageIQ/manageiq-api#3 (comment)) and fulfilled [by ManageIQ here](https://github.com/ManageIQ/manageiq/pull/15799), a metadata function that should return the description of the data stored in the options field was added!
+Cross repo collaboration is the name of our game.
+### Operational Alerts Table Documentation
+Can not forget da docs!  Operational alerts table docs were improved [here](https://github.com/ManageIQ/manageiq_docs/pull/486_
```

Comment:
Oh oops looks like there's another s/_/)/ here to make that link work. Think that's the last one!